central/cve/image/v2/datastore/store/postgres/gen.go

```diff
@@ -0,0 +1,3 @@
+package postgres
+
+//go:generate pg-table-bindings-wrapper --type=storage.ImageCVEV2 --table=image_cves_v2 --references=storage.Image,storage.ImageComponentV2 --search-category IMAGE_VULNERABILITIES_V2 --search-scope IMAGE_VULNERABILITIES_V2,IMAGE_COMPONENTS_V2,IMAGES,DEPLOYMENTS,NAMESPACES,CLUSTERS --feature-flag FlattenCVEData
```

Comment:
Same comment as image components v2. Do you need `IMAGE_VULNERABILITIES_V2` in `search-scope` too?

Reply:
same response as on components